### PR TITLE
Random tile, text palette, ITCM functions

### PR DESCRIPTION
**Pull request by assidion2**

### CODE REVIEW

File: symbols/arm9.yml [hunk 1]
```diff
@@ -5070,6 +5070,18 @@ arm9:
         r1: x offset within window
         r2: y offset within window
         r3: text to draw
+    - name: GetColorCodePaletteOffset
+      address:
+        EU: 0x20268A8
+        NA: 0x20265C4
+        JP: 0x202669C
+      description: |-
+        Gets the offset of a text color symbol's 2-byte RGB5 color in the palette stored in VRAM at 0x6882000.
```
Comment (UsernameFodder) [inline symbols/arm9.yml:5079]: Is this VRAM offset the same for all versions?

File: headers/functions/arm9/itcm.h [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #ifndef HEADERS_FUNCTIONS_ARM9_ITCM_H_
 #define HEADERS_FUNCTIONS_ARM9_ITCM_H_
 
+void CopyAndInterleave(int* dst, int* src, int len, int val);
```
Comment (UsernameFodder) [inline headers/functions/arm9/itcm.h:4]: Are these really 4-byte signed `int` values? The description makes me think `dst` should be `uint32_t*`, `src` should be `uint16_t*`, and `val` should be `uint8_t`, though I haven't looked at what the access pattern looks like.

File: headers/functions/arm9/itcm.h [hunk 3]
```diff
@@ -26,6 +27,7 @@ int GetKeyM2N(enum monster_id monster_id);
 int GetKeyM2NBaseForm(enum monster_id monster_id);
 void HardwareInterrupt(void);
 void ReturnFromInterrupt(void);
+void InitDmaTransfer_Standard(int channel, int* src, int* dst, uint16_t word_count);
```
Comment (UsernameFodder) [inline headers/functions/arm9/itcm.h:30]: Should the pointers be `void*`? DMA seems more generic than specifically integers

File: headers/functions/arm9/itcm.h (hunk 2 above)
Comment (assidion2) [reply headers/functions/arm9/itcm.h:4]: The accesses are all ldrh/strh so I suppose both should be uint16_t*

File: symbols/arm9.yml (hunk 1 above)
Comment (UsernameFodder) [reply symbols/arm9.yml:5079]: Ok seems reasonable

**Review by UsernameFodder — APPROVED** [review]
Thanks!

**Pull request merged**